shell/platform/windows/window_win32.cc

```diff
@@ -61,6 +63,10 @@ WindowWin32::WindowWin32(
   // supported, |current_dpi_| should be updated in the
   // kWmDpiChangedBeforeParent message.
   current_dpi_ = GetDpiForHWND(nullptr);
+
+  // Get initial value for wheel scroll lines
```

Comment:
Would you mind leaving a TODO here to listen to scroll speed setting changes? Something like:

```suggestion
  // Get initial value for wheel scroll lines
  // TODO: Listen to changes for this value
  // https://github.com/flutter/flutter/issues/107248
```

Reply:
Done